wrappers/python/README.md

```diff
@@ -6,27 +6,62 @@
 ## Installation
 
 ```bash
-pip install zxing-cpp
+pip install --upgrade zxing-cpp
 ```
+
 or
 
 ```bash
 python setup.py install
 ```
 
-[Note: To install via `setup.py` (or via `pip install` in case there is no pre-build wheel available for your platfor or python version), you need a suitable [build environment](https://github.com/zxing-cpp/zxing-cpp#build-instructions) including a c++ compiler.]
```

Comment:
The `[Note: ...]` brackets are a stylistic choice used in other readmeas as well. Please leave them alone.